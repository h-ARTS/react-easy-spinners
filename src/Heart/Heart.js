import React from 'react'
import './styles.css'

const Heart = (props) => (
  <>
    <div className="lds-heart">
      <div></div>
    </div>
  </>
)

export default Heart
