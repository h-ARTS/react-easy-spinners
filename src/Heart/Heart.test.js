import React from 'react'
import renderer from 'react-test-renderer'
import Heart from './Heart'

describe('Heart component', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Heart />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
