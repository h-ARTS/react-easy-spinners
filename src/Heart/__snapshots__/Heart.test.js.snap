// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heart component should render correctly 1`] = `
<div
  className="lds-heart"
>
  <div />
</div>
`;
